ledger/blockdb.go

```diff
@@ -208,6 +222,22 @@ func blockLatest(tx *sql.Tx) (basics.Round, error) {
 	return 0, fmt.Errorf("no blocks present")
 }
 
+// Returns amount of blocks in Block DB
+// At least one row (genesis block) must present, empty table is threated as error
```

Comment:
This is pretty expensive, and will cause algod to take a very long time to start.

How about using the min round number to check for archival?  `SELECT MIN(rnd) FROM blocks` should be much faster, and if that's 0, you have all of the blocks.

Reply:
@tsachiherman 
Measured performance with 13565K blocks in the DB (roughly equals to blocks from 2 years of runtime).
COUNT(1):       13,564,192 - 3m34.966600652s
COUNT(rnd):   13,564,192 - 3m36.368089268s

Table without deletion:
MIN(rnd):  387,385 - 2.120839ms

Table after  ~5M records deleted:
MIN(rnd):  5,000,000 - 1.201572ms

Reply:
Done